### ISSUE TITLE
This application could use a dark mode

### ISSUE THREAD

**Issue by KacperWieczorek** [reportportal/reportportal#1575]
**Describe the solution you'd like**
With the prevalence of dark mode in software used by many programmers it's a shame that Report Portal does not offer dark mode.

**Comment by evjlobanova** [comment]
It is in a roadmap for 2022

**Comment by itzika** [comment]
what about dark mode?

**Comment by Kozovskaya** [comment]
Hello @itzika @KacperWieczorek!
Unfortunately the dark mode is not in place yet. The priorities are constantly changing and the dark mode is not on the top.

**Comment by frasu** [comment]
Maybe in 2025 is a chance for the Dark Mode?

**Comment by Kozovskaya** [comment]
Hello @frasu,
Unfortunately, the priorities are different. Not in 2025

**Comment by YoussefMehany** [comment]
Maybe there is a chance for the Dark Mode in 2026?

**Comment by fslev** [comment]
Yeah, current light theme has become somewhat boring and also too bright while suddenly switching from dark applications.

**Comment by hrworksImm** [comment]
A darkmode would be much appreciated 🕶️
